tests/testDipoleFitter.py

```diff
@@ -55,17 +56,25 @@ class DipoleFitTestGlobalParams(object):
     @var offsets: pixel coordinates between lobes of dipoles
     """
 
-    def __init__(self, xc=np.array([65.3, 24.2]), yc=np.array([38.6, 78.5])):
-        """Initialize the parameters."""
+    def __init__(self, xc=None, yc=None):
+        """Initialize the parameters.
+
+        @param xc iterable x coordinate (pixels) of center(s) of input dipole(s)
+        @param yc iterable y coordinate (pixels) of center(s) of input dipole(s)
+        """
         np.random.seed(666)
         self.display = False
         self.verbose = False
         self.w, self.h = 100, 100  # size of image
 
         self.xc = xc  # xcenters of two dipoles in image
+        if xc is None:
+            xc = self.xc = [65.3, 24.2]
```

Comment:
Why do you have xc and yc here? They aren't used for anything, just self.xc/self.yc. One option:

`self.xc = xc if xc is not None else [65.3, 24.2]`